parm/config/gfs/config.ufs

```diff
@@ -281,7 +281,7 @@ case "${fv3_res}" in
           export rf_cutoff=100.0
           export fv_sg_adj=450
           export WRITE_GROUP_GDAS=2
-          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=10
+          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE_GDAS=15
```

Comment:
@junwang-noaa might have a recommendation for what some of the settings should be for this resolution, but it can be addressed in a separate PR.